windows/Gyroscope.cs

```diff
@@ -29,20 +30,24 @@ private void ReadingChanged(object sender, Sensors.GyrometerReadingChangedEventA
         {
             Sensors.GyrometerReading reading = e.Reading;
 
-            this.SendEvent("Gyroscope", new RNSensorsJsonObject
+            if (_lastReading.AddMilliseconds(interval) <= reading.Timestamp)
```

Comment:
I don't think, see my comment on the PR, it seems the driver sensors of uwp determine itself which is the best updateInterval to set, regarding all the process which are using this driver, and send event at this interval.

Seems to be the same with the android part... :/
Please look at Issue #12 & PR #15